portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,214 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="utf-8" />
```

Comment:
no need to close meta tags as these are considered self-closing "void" tags. this should be `<meta charset="utf-8">`

https://google.github.io/styleguide/htmlcssguide.html#Document_Type